test/schema_envelop.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE subscribers (
+  email VARCHAR(256) NOT NULL,
+  address VARCHAR(256) NOT NULL,
+  subscriptions JSON DEFAULT NULL,
+  created BIGINT NOT NULL,
+  verified BIGINT NOT NULL DEFAULT 0,
+  PRIMARY KEY (email, address),
```

Comment:
The primary key can be just `address` no?  because a user cannot use multiple emails. this will help us to use queries like `INSERT ... ON DUPLICATE KEY UPDATE` to insert or update email subscription in   `src/writer/email-subscription.ts`

Reply:
An address can be associated to multiple email addresses, but only one can be verified.

I think this was done so an unverified address can subscribe again with another email address